docs/UserGuide.md

```diff
@@ -324,6 +325,35 @@ Examples:
 
 ![result for 'edit event'](images/editEventResult.png)
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Sorting the patient list : `sort`
+
+Sorts the patient list in the address book by specified attribute.
+
+Format: `sort [ATTRIBUTE]`
+
+* Sorts all the added patients in the patient list by specified `ATTRIBUTE`.
+* Supported `ATTRIBUTE`s include `n` for patient name and `p` for patient's preferred name.
+* There can only be 1 or 0 `ATTRIBUTE`.
```

Comment:
DG uses one and zero, 
could consider standardising the UG and DG to one of the options

Reply:
Good catch! will proceed to standardise to one or zero!